### PR TITLE
Add info about nightly and weekly DALI builds

### PR DESCRIPTION
**Pull request by JanuszL**
Signed-off-by: Janusz Lisiecki <jlisiecki@nvidia.com>

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -105,6 +105,46 @@ Installing this package will install ``nvidia-dali`` and its dependencies, if th
     OLDER_VERSION=0.6.1
     pip install --extra-index-url https://developer.download.nvidia.com/compute/redist nvidia-dali-tf-plugin==$OLDER_VERSION
 
+Nightly and weekly release channels
+"""""""""""""""""""""""""""""""""""
+
+.. note::
+
+  While binaries available to download from nightly and weekly build includes most recent changes available in the GitHub some functionalities may not work or provide inferior performance comparing to the official releases. Those builds are meant for the early adopters seeking for the most recent version available and being ready to explore the unknown.
```
Comment (awolant) [inline README.rst]: I think `include` instead of `includes`

File: README.rst [hunk 2]
```diff
@@ -105,6 +105,46 @@ Installing this package will install ``nvidia-dali`` and its dependencies, if th
...
+
+.. code-block:: bash
+
+  pip install --extra-index-url https://developer.download.nvidia.com/compute/redist/nightly/cuda/10.0 nvidia-dali-nightly
+  pip install --extra-index-url https://developer.download.nvidia.com/compute/redist/nightly/cuda/10.0 nvidia-dali-tf-plugin-nightly
+
+Weekly builds
+**************
+
+Also, there is a weekly release channel with more thorough testing (only CUDA10 build are provided there):
```
Comment (awolant) [inline README.rst]: I think `builds` instead of `build`

File: README.rst [hunk 3]
```diff
@@ -373,3 +413,4 @@ DALI was built with major contributions from Trevor Gale, Przemek Tredak, Simon
 
 .. |Documentation| image:: https://img.shields.io/badge/Nvidia%20DALI-documentation-brightgreen.svg?longCache=true
    :target: https://docs.nvidia.com/deeplearning/sdk/dali-developer-guide/
+ fo
```
Comment (jantonguirao) [inline README.rst]: what is this?

Comment (JanuszL) [reply README.rst]: Random leftover, removed.

**Pull request merged**